llvm/test/CodeGen/AMDGPU/preload-implicit-kernargs-debug-info.ll

```diff
@@ -1,7 +1,7 @@
-; RUN: not --crash opt -mtriple=amdgcn-amd-amdhsa -mcpu=gfx940 -passes='amdgpu-attributor,function(amdgpu-lower-kernel-arguments)' -amdgpu-kernarg-preload-count=16 -S < %s 2>&1 | FileCheck %s
+; RUN: opt -mtriple=amdgcn-amd-amdhsa -mcpu=gfx940 -passes='amdgpu-attributor,function(amdgpu-lower-kernel-arguments)' -amdgpu-kernarg-preload-count=16 -S < %s 2>&1 | FileCheck %s
 
-; CHECK: function declaration may only have a unique !dbg attachment
-; CHECK-NEXT: ptr @0
+; CHECK: define amdgpu_kernel void @preload_block_count_x{{.*}} !dbg ![[#]]
+; CHECK-NOT: declare void @0{{.*}} !dbg ![[#]]
```

Comment:
This is much too fragile for a -NOT check, use generated positive checks?

Reply:
Do you happen to know of how to get generated checks to include an unreferenced declare? I suppose in general some extra declares are benign, but in this case it is the primary thing being tested.

I've tried `update_test_checks.py --check-globals all --include-generated-funcs` but no luck

Reply:
I don't think update_test_checks has a way to do this

Reply:
Might as well go ahead with this as a temporary step, it at least adds the test

Reply:
I guess you could try adding -implicit-check-not=declare to FileCheck